### PR TITLE
fix: bsc rate limit for alm

### PR DESCRIPTION
**Pull request by t0mcr8se**

### CODE REVIEW

File: alm/.env.example [hunk 1]
```diff
@@ -1,15 +1,16 @@
-COMMON_HOME_BRIDGE_ADDRESS=0x2CA5411c4bf447Cc27CD6E6d1d046f922A27C399
-COMMON_FOREIGN_BRIDGE_ADDRESS=0x63C47c296B63bE888e9af376bd927C835014039f
+COMMON_HOME_BRIDGE_ADDRESS=0x1ee6E3E3d2DE779858728E157B3B9C488bA7b706
+COMMON_FOREIGN_BRIDGE_ADDRESS=0x3A5A320a2f98a3Fe39c9040e7e3E9caA7F0D5bd6
 
-COMMON_HOME_RPC_URL=https://rpc.fuse.io
-COMMON_FOREIGN_RPC_URL=https://mainnet.infura.io/v3/
+COMMON_HOME_RPC_URL=https://rpc.fuse.io/
+COMMON_FOREIGN_RPC_URL=https://red-restless-morning.bsc.quiknode.pro/<quicknode-project-id>/
```
Comment (leonprou) [inline alm/.env.example]: please delete the quicksart  RPC.

Comment (leonprou) [reply alm/.env.example]: if there any requirements to the RPC, let's mention them in a comment

File: alm/Dockerfile [hunk 2]
```diff
@@ -19,7 +19,7 @@ COPY --from=contracts /mono/contracts/build ./contracts/build
 COPY commons/package.json ./commons/
 COPY alm/package.json ./alm/
 COPY yarn.lock .
-RUN NOYARNPOSTINSTALL=1 yarn install --frozen-lockfile --production
+RUN NOYARNPOSTINSTALL=1 yarn install --frozen-lockfile
```
Comment (leonprou) [inline alm/Dockerfile:22]: you know why they deleted the --`production` flag?

**Review by leonprou — CHANGES_REQUESTED** [review]

File: alm/.env.example (hunk 1 above)
Comment (t0mcr8se) [reply alm/.env.example]: We need to use the quicknode rpc, although it has limit on the eth_getlogs method, it is still faster than the commented bsc node

File: alm/Dockerfile (hunk 2 above)
Comment (t0mcr8se) [reply alm/Dockerfile:22]: not sure why, it was changed in this commit https://github.com/poanetwork/tokenbridge/commit/818bc4675d24b9bf1823e4979c6af4e3baf62825#diff-5fa3649758ff1fea6001a52abbc8b6f3410fbdbf30c4137a776f6431de6ebf29

Comment (t0mcr8se) [reply alm/Dockerfile:22]: @leonprou do you think I should change it back to `--production` ?

Comment (leonprou) [reply alm/Dockerfile:22]: mmm I actually don't care I build the files and host this on S3.
If that's what POA did, let's leave it. I though maybe you changed this for debugging purposes

File: alm/.env.example (hunk 1 above)
Comment (leonprou) [reply alm/.env.example]: Alright.
But can you move the BSC changes to `.env.bsc.example` file. And keep `.env.example` untouched

**Review by leonprou — APPROVED** [review]

**Pull request merged**